usr.sbin/inetd/parse_v2.h

```diff
@@ -0,0 +1,10 @@
+#include "inetd.h"
+
+typedef enum parse_v2_result {V2_SUCCESS, V2_SKIP, V2_ERROR} parse_v2_result;
```

Comment:
Perhaps fold parse_v2.h this into inetd.h?

Reply:
We had it in inetd.h up until a couple days ago when I separated them because I thought it might be more correct to have a separate parse_v2.h :D. It does feel a little silly to have such a small header. I'll merge them back.